### PR TITLE
Update generated code in `pkg/nodegroup/` and improve the build

### LINKED ISSUE #1229: Missing m5.16xlarge from max_pods.map due to which kubelet fails to start
**What happened?**
A description of actual behavior (with error messages).
Missing m5.16xlarge from max_pods.map due to which kubelet fails

**What you expected to happen?**

Missing m5.16xlarge instance type from the max_pods.map is causing kubelet daemon not able to start issue. 

**How to reproduce it?**
Include the steps to reproduce the bug
Launch m5.16xlarge instance type which would fail to join the cluster. If you check kubelet logs, you would see below error:

```
Aug 29 18:58:50 ip-192-168-5-5.us-west-2.compute.internal kubelet[25640]: F0829 18:58:50.172283   25640 server.go:147] invalid argument "" for "--max-pods" flag: strconv.ParseInt: p
```

### PR DESCRIPTION
**Pull request by errordeveloper**
### Description

- Fix #1229 
- Improve makefile
- Stop relying on asset permissions

### Checklist

- [x] Code compiles correctly (i.e `make build`)
- [x] Added tests that cover your change (if possible)
- [x] All unit tests passing (i.e. `make test`)
- [x] Manually tested

### CODE REVIEW

**Comment by errordeveloper** [comment]
@cPu1 thanks!

File: pkg/nodebootstrap/userdata.go [hunk 1]
```diff
@@ -51,20 +46,19 @@ func addFilesAndScripts(config *cloudconfig.CloudConfig, files configFiles, scri
 				Path: dir + fileName,
 			}
 			if file.isAsset {
-				data, info, err := getAsset(fileName)
+				data, err := getAsset(fileName)
 				if err != nil {
 					return err
 				}
 				f.Content = data
-				f.Permissions = fmt.Sprintf("%04o", uint(info.Mode()))
```
Comment (errordeveloper) [inline pkg/nodebootstrap/userdata.go]: We didn't actually use it for anything, as it posed a challenge on the build, I figured there is no point of keeping this (untested) feature.

File: pkg/nodebootstrap/assets.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ func _10EkscltAl2Conf() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "10-eksclt.al2.conf", size: 999, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "10-eksclt.al2.conf", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
```
Comment (cPu1) [inline pkg/nodebootstrap/assets.go]: Where do we use `asset.info`? I could only find `asset.bytes` being used but not `info`

Comment (errordeveloper) [reply pkg/nodebootstrap/assets.go]: It was used It's no longer used for setting remote permissions, but we didn't make use of it in a long time, hence it's being killed in this PR as it made the build messy - as local permissions cannot be relied on... (well git track's owner's bit, but not group's or others').

Comment (cPu1) [reply pkg/nodebootstrap/assets.go]: I see, thanks for the info. We should plan to remove having this field completely.

Comment (cPu1) [reply pkg/nodebootstrap/assets.go]: Looks like go-bindata doesn't support making these fields optional.

**Pull request merged**